_includes/codepens/full-featured/index.js

```diff
@@ -235,6 +235,7 @@ tinymce.init({
   },
   menubar: 'file edit view insert format tools table tc help',
   toolbar: 'undo redo | bold italic underline strikethrough | fontselect fontsizeselect formatselect | alignleft aligncenter alignright alignjustify | outdent indent |  numlist bullist checklist | forecolor backcolor casechange permanentpen formatpainter removeformat | pagebreak | charmap emoticons | fullscreen  preview save print | insertfile image media pageembed template link anchor codesample | a11ycheck ltr rtl | showcomments addcomment',
+  toolbar_sticky: true,
```

Comment:
Isn't this going to make the header go behind the banner at the top of the page?

Reply:
Yes. I would like to keep this in case we change the site to prevent this issue, but i have realised that if they have a menu open it will stick out from under the header. Not sure if I should leave or remove.

Reply:
I'd vote to remove it, or at the very least comment it out.

Reply:
removed